examples/3d_examples/OctopusSwimming/run_tapered_arm_stroke.py

```diff
@@ -0,0 +1,462 @@
+import numpy as np
+from sopht.utils.precision import get_real_t
+from set_environment_tapered_arm import Environment
+import sopht.simulator as sps
+import elastica as ea
+import sopht.utils as spu
+import click
+from octopus_initializer_functions import (
+    assemble_octopus,
+    initialize_activation_functions,
+)
+
+
+def tapered_arm_and_cylinder_flow_coupling(
```

Comment:
shouldnt this be called more informative? like swimming

Reply:
4ce37ff